vcr/types/interface.go

```diff
@@ -117,6 +116,8 @@ type Resolver interface {
 
 // VCR is the interface that covers all functionality of the vcr store.
 type VCR interface {
+	Issuer() issuer.Issuer
```

Comment:
maybe VCR should just implement `Issuer`  and delegate the calls? Now it's used as sort of "service registry" (just a holder of services to be used), while we normally call business functions on engine types (instead of getting one of its dependencies, and then call functions on it).

Reply:
We could do that, but mind there will be a holder and issuer as well. This will be a lot of "empty" functions. We can also make these separate engines, but that;s also a bit weird. So, this is what I settled on. But the inclusion of the issuer.Issuer causes import cycles. So this is not optima.